share/spice/isslocation/isslocation.js

```diff
@@ -0,0 +1,37 @@
+(function(env) {
+    "use strict";
+    env.ddg_spice_isslocation = function(api_result) {
+        
+       if(!api_result || api_result !== "success") {
+            return Spice.failed('isslocation');
+        }
+        var issLatitude = (Math.round(api_result.iss_position.latitude * 100) / 100).toFixed(2);
+        var issLongitude = (Math.round(api_result.iss_position.longitude * 100) / 100).toFixed(2);
+    
+        DDG.require('maps', function()
+        Spice.add({
+            id: "isslocation",
+            name: "ISSLocation",
+            model: 'Place',
+
+            view: 'Map',
+            data: [{
```

Comment:
@jagtalon How does that work? Do you mean I should add 

url: "(link goes here)",